src/ConnectCore.js

```diff
@@ -107,6 +133,18 @@ class ConnectCore {
   }
 
   /**
+   *  Subscribe to different events
+   */
+
+   subscribe(topic) {
```

Comment:
don't necessarily disagree either, but thought it was worth introducing with the expectation that we will add more plugins/buttons etc and will want a model for listening to different events on those components. Pub/sub is widely used and in things like the Facebook js sdk for their plugins, would not expect the overhead to be to great especially with the simple copy/paste examples and will easily allow adding additional events to plugins, besides the one right now (on click, on auth change, on logout, etc) (needs some naming conventions as well). But yes of course a single onAuthChange() function could just return a promise, but events and plugins button may expand quickly and introduce breaking changes in the future if we changed the model.